Workflow.pkl

```diff
@@ -490,65 +413,412 @@ typealias WorkflowRunType =
+  ///
+  /// Note: The webhook payload available to GitHub Actions does not include the added, removed, and modified attributes
+  /// in the commit object.
+  /// You can retrieve the full commit object using the REST API.
+  /// For more information, see <https://developer.github.com/v3/repos/commits/#get-a-single-commit.>
+  ///
+  /// See <https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push>
   push: Push?
+
+  /// Runs your workflow anytime a package is published or updated.
+  ///
+  /// For more information, see <https://help.github.com/en/github/managing-packages-with-github-packages>.
+  ///
+  /// See <https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#registry_package>
   registry_package: RegistryPackage?
+
+  /// Runs your workflow anytime the release event occurs.
+  ///
+  /// More than one activity type triggers this event.
+  /// For information about the REST API, see <https://developer.github.com/v3/repos/releases/> in the GitHub Developer documentation.
+  ///
+  /// See <https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release>
   release: Release?
+
+  /// You can use the GitHub API to trigger a webhook event called `repository_dispatch` when you want to trigger a
+  /// workflow for activity that happens outside of GitHub.
+  ///
+  /// For more information, see <https://developer.github.com/v3/repos/#create-a-repository-dispatch-event>.
+  /// To trigger the custom `repository_dispatch` webhook event, you must send a POST request to a GitHub API endpoint
+  /// and provide an `event_type` name to describe the activity type.
+  /// To trigger a workflow run, you must also configure your workflow to use the `repository_dispatch` event.
+  ///
+  /// See <https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#repository_dispatch>
   repository_dispatch: RepositoryDispatch?
-  schedule: Schedule?
-  status: Status?
-  watch: Watch?
+
+  /// You can schedule a workflow to run at specific UTC times using POSIX cron syntax
+  /// (https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07).
+  ///
+  /// Scheduled workflows run on the latest commit on the default or base branch.
+  /// The shortest interval you can run scheduled workflows is once every 5 minutes.
+  /// Note: GitHub Actions does not support the non-standard syntax @yearly, @monthly, @weekly, @daily, @hourly,
+  /// and @reboot.
+  /// You can use crontab guru (<https://crontab.guru/>) to help generate your cron syntax and confirm what time it will
+  /// run.
+  /// To help you get started, there is also a list of crontab guru examples (<https://crontab.guru/examples.html>).
+  ///
+  /// See <https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule>
+  schedule: Listing<Schedule>?
```

Comment:
Bugfix: `schedule` should be a YAML sequence (and therefore a `Listing`); see https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#onschedule

Reply:
I investiaged in it a bit and think that `pkl` rendere`yml` correctly, also in a way GitHub Action can understand it!
There is an "issue" with a leading `*`(asteriks) in `yml`. This is another derictive.
So in case a cron starts with that, it should be quoted. Otherwise it can be left out❗ 
So the outcome is 100% fine.

I also used the [actionlint](https://rhysd.github.io/actionlint/#eJx1kUFrgzAUx+/5FO8wEAquHawXTyubSBlY0Y3Rk6TZA7vFxOUlXkq/+9RUuraOHELy+//yXniK1xhBjhI5IVgNNW9RgUBlDZdAijdUaUuMaRUxgMZR1e8AO8OVqJD8CSCEoOZ7FQwhKUuDPw7J9pi63KeTeI4K0z0HwWz+sICZX8EVXMKjBzcSeGcRMPald0MDxn/AFzBOUdhHay40hZLbro+BoGrHHpIsKYt1kq7TpHyNtxHcHQ5AKAxaur+CcDxOWNmqKD42+cv/6pg4+8UmXb1ts7h8L+L8UrxAE8Z0uRs8mmSx+TMbNcz5uULxrZ09XQM46gYIXNi9VjQXJ/zULtkvl4WX8w==) to verify.